### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by evertonrodrigues**
Adding clients with reactjs and vuejs, and servers with nodejs and java

### CODE REVIEW

File: server_node/config.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "platformUrl":"https://iotmid-docker42.cpqd.com.br",   
+    "headers": {
+        "fiware-service": "admin",
+        "fiware-servicepath": "/"
+    },
+    "security": {        
+        "username":"admin",
+        "passwd":"admin"
```
Comment (mmagr) [inline server_node/config.json]: same here

File: server_java/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+server.port=3001
+wss.server.port=9092
+wss.server.host=localhost
+
+platform.url=https://iotmid-docker42.cpqd.com.br
```
Comment (mmagr) [inline server_java/src/main/resources/application.properties]: I have a feeling we should keep this invalid in source code, and have a note somewhere in README.md telling users to update this with a valid url.

File: client_react/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "app-root-path": "^2.0.1",
+    "gulp-sass": "^3.1.0",
+    "react-scripts": "0.8.5"
+  },
+  "dependencies": {
+    "chart.js": "^2.5.0",
+    "jquery": "^2.1.4",
+    "materialize-css": "^0.97.3",
+    "moment": "^2.18.1",
+    "mqtt": "^2.2.1",
```
Comment (mmagr) [inline client_react/package.json]: this is not being used, right?

File: server_node/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "ssh://git@stash.atlantico.com.br/iot/iot_blank_project.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.17.2",
+    "express": "^4.15.3",
+    "mqtt": "^2.8.0",
```
Comment (mmagr) [inline server_node/package.json]: this is not being used right?

File: server_node/config.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+    "platformUrl":"https://iotmid-docker42.cpqd.com.br",
```
[the remark this answers is not in the document]
Comment (evertonrodrigues) [reply server_node/config.json]: I agree with you. I will going to change theses things.

**Comment by evertonrodrigues** [comment]
I agree with you. I will going to change theses things.

**Pull request merged**